apps/projects/serializers.py

```diff
@@ -332,3 +332,30 @@ def get_moderation_detail_url(self, instance):
         return reverse(
             "userdashboard-moderation-detail", kwargs={"slug": instance.slug}
         )
+
+
+class ModeratorProjectSerializer(ProjectSerializer):
+    num_reports = serializers.SerializerMethodField()
```

Comment:
I think we only use the ProjectSerializer for the Moderation Dashboard, so you could just change that one. But renaming it is actually a good idea!

Reply:
merged and renamed it